x/auth/types/account.go

```diff
@@ -138,13 +139,20 @@ func (acc BaseAccount) Validate() error {
 }
 
 func (acc BaseAccount) String() string {
-	out, _ := acc.MarshalYAML()
+	out, err := acc.MarshalYAML()
+	if err != nil {
+		panic(err)
+	}
+
 	return out.(string)
 }
 
 // MarshalYAML returns the YAML representation of an account.
 func (acc BaseAccount) MarshalYAML() (interface{}, error) {
```

Comment:
```suggestion
func (acc BaseAccount) marshalYAML() (interface{}, error) {
```
should this be private?